src/key.ts

```diff
@@ -1,2 +1,2 @@
-export const api_key = '250b65ac19c1d23b70718726a42ee160'
+export const apiKey = '250b65ac19c1d23b70718726a42ee160'
```

Comment:
We eventually want to remove the API key from the source code (it's not a big deal in this particular project), so you could consider creating an Issue for this, to track the work.

Reply:
Sounds good, and I agree, I was thinking that it should be away to add it as an environment variable.